packages/flutter/lib/src/widgets/async.dart

```diff
@@ -498,6 +588,7 @@ class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
 /// ```dart
 /// FutureBuilder<String>(
 ///   future: _calculation, // a previously-obtained Future<String> or null
```

Comment:
we should make it clearer that _calculation _must not be created during the call to the build method_ that creates the `FutureBuilder`.